back/spec/models/custom_field_spec.rb

```diff
@@ -523,8 +523,8 @@ def visit_shapefile_upload(_field)
       expect(other_option_text_field).not_to be_nil
       expect(other_option_text_field.key).to eq 'select_field_other'
       expect(other_option_text_field.input_type).to eq 'text'
-      expect(other_option_text_field.title_multiloc['en']).to eq "If you picked 'Something else', what are you thinking of?"
-      expect(other_option_text_field.title_multiloc['fr-FR']).to eq 'Puisque vous avez choisi « Quelque chose », à quoi pensez-vous ?'
+      expect(other_option_text_field.title_multiloc['en']).to eq 'Type your answer'
+      expect(other_option_text_field.title_multiloc['fr-FR']).to eq 'Tapez votre réponse'
```

Comment:
Jus a note:

I like that you test one other language, but I think this will fail when you merge to master, as I think CI will remove the fr-FR.yml file.

You could comment out this line, then try to remember to merge a 'fix' after the translations are ready and have been merged into master from `l10n_master`